node/bft/src/bft.rs

```diff
@@ -554,6 +566,12 @@ impl<N: Network> BFT<N> {
                 dag_write.commit(certificate, self.storage().max_gc_rounds());
             }
         }
+        // Update the last election certificate IDs.
+        {
+            let mut last_election_certificate_ids = self.last_election_certificate_ids.write();
+            last_election_certificate_ids.clear();
+            last_election_certificate_ids.extend(election_certificate_ids);
```

Comment:
```suggestion
            *last_election_certificate_ids = election_certificate_ids;
```
this should be possible